### PR TITLE
Catch duplicate account request errors

### PR DESCRIPTION
**Pull request by chriskuehl**

### CODE REVIEW

File: ocflib/account/submission.py [hunk 1]
```diff
@@ -197,14 +198,21 @@ def validate_then_create_account(request):
...
 
-                with get_session() as session:
-                    session.add(stored_request)  # TODO: error handling
-                    session.commit()
+                try:
+                    with get_session() as session:
+                        session.add(stored_request)  # TODO: error handling
+                        session.commit()
+                except sqlalchemy.exc.IntegrityError:
+                    # If there's an integrity constaint, it's okay -- the
```
Comment (matthew-mcallister) [inline ocflib/account/submission.py]: "constaint"

Comment (jvperrin) [reply ocflib/account/submission.py]: Pretty sure I would have just read over that without noticing

Comment (chriskuehl) [reply ocflib/account/submission.py]: typing is hard

Comment (matthew-mcallister) [reply ocflib/account/submission.py]: @jvperrin Yeah, I guess could have ignored that since it's only a comment and not published anywhere.

**Pull request merged**